adoc-mode.el

```diff
@@ -56,36 +55,38 @@
 Based upon AsciiDoc version 8.5.2. I.e. regexeps and rules are
 taken from that version's asciidoc.conf / manual.")
 
-
 ;;;; customization
-(defgroup adoc nil
-  "Major-mode for editing AsciiDoc files in Emacs.
-
-Most faces adoc-mode uses belong to the markup-faces
-customization group, see link below, and have to be customized
-there.  adoc-mode has only a few faces of its own, which can be
-customized on this page."
+(defgroup adoc nil "Support for editing AsciiDoc files in GNU Emacs."
   :group 'text
   :prefix "adoc-"
+  :version "0.7.0"
   :link '(url-link "https://github.com/bbatsov/adoc-mode"))
 
+
+(defcustom adoc-mode-hook nil
```

Comment:
I think this hook automatically defined by the major mode, so it's not really needed as an explicit variable.

Reply:
Removed.